internal/store/static.go

```diff
@@ -114,20 +114,33 @@ func (store *StaticStore) GetPlugins(pluginFilter *model.PluginFilter) ([]*model
 }
 
 // getPlugins returns all plugins compatible with the given server version, sorted by name ascending.
-func (store *StaticStore) getPlugins(serverVersion string) ([]*model.Plugin, error) {
+func (store *StaticStore) getPlugins(serverVersion string, includeEnterprisePlugins bool) ([]*model.Plugin, error) {
 	var result []*model.Plugin
 	plugins := map[string]*model.Plugin{}
 
 	for _, storePlugin := range store.plugins {
-		if serverVersion != "" && storePlugin.Manifest.MinServerVersion != "" {
-			meetsMinServerVersion, err := storePlugin.Manifest.MeetMinServerVersion(serverVersion)
+		if serverVersion != "" {
+			sv, err := semver.Parse(serverVersion)
 			if err != nil {
-				return nil, errors.Wrapf(err, "failed to check minServerVersion for manifest.Id %s", storePlugin.Manifest.Id)
+				return nil, errors.Wrapf(err, "failed to parse serverVersion %s", serverVersion)
 			}
 
-			if !meetsMinServerVersion {
+			// Workaround for https://mattermost.atlassian.net/browse/MM-26507
```

Comment:
Mind putting another comment something like: 
`// Honour enterprise flag for server version >= 5.25.0 only.`
just for quicker understanding of why we need to do this.

Reply:
:+1: Added